sdk/core/System.ClientModel/tests/client/TestClients/SimpleClient/SimpleClientServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace ClientModel.ReferenceClients.SimpleClient;
+
+public static class SimpleClientServiceCollectionExtensions
+{
+    // TODO: How much of these can SCM provide for SCM-based clients to use
+    // from a central source?  Microsoft.Extensions.ClientModel, e.g.?
```

Comment:
TODO: It would be good to figure out if we can centralize this without having to add all the abstraction layers/complexity we have in Microsoft.Extensions.Azure